checker/manual/external-tools.tex

```diff
@@ -324,60 +324,90 @@ \section{Maven\label{maven}}
             <artifactId>checker-qual</artifactId>
             <version>\ReleaseVersion{}</version>
         </dependency>
-
+        <dependency>
+            <groupId>org.checkerframework</groupId>
+            <artifactId>checker</artifactId>
+            <version>\ReleaseVersion{}</version>
+        </dependency>
+        <!-- The annotated JDK to use (change to jdk7 if using Java 7) -->
+        <dependency>
+            <groupId>org.checkerframework</groupId>
+            <artifactId>jdk8</artifactId>
+            <version>\ReleaseVersion{}</version>
+        </dependency>
```

Comment:
Need one more dependency: a dependency on the 'compiler' artifact (same groupId: org.checkerframework, same version).